### ISSUE TITLE
Altair Themes Ignored in PDF Output

### ISSUE THREAD

**Issue by EntilZha** [altair-viz/altair_saver#81]
I've been debugging an issue where it seems like altair is not preserving the theme for PDF output (e.g., fivethirtyeight).

I poked around in a debug session and think I understand what is going on, but not certain.

For SVG/Json/Html/Png, the theme option is being respected since they use vega-embed (png seems to use selenium, so html which uses vega embed). However, the vega CLI tools don't seem to use vega embed and don't otherwise handle theming. Someone reported a similar issue on stackoverflow, but with the png CLI tool https://stackoverflow.com/questions/59973181/how-to-apply-a-vega-theme-when-using-vl2png

Any suggestions on a workaround within altair to fix this up? The post on SO seems like it would work, but is a bit inconvenient.

**Comment by jakevdp** [comment]
PDF output happens via vega-cli, which unfortunately does not directly support themes. See this StackOverflow answer for a workaround when using the CLI itself: https://stackoverflow.com/questions/59973181/how-to-apply-a-vega-theme-when-using-vl2png

I don't know of any examples of doing this from Python using altair-saver.

I would suggest avoiding themes and setting chart properties directly.

**Comment by EntilZha** [comment]
I'm pretty I linked the same workaround ;)

I ended up doing a similar workaround where I exported the theme as a json, converted it to a python dictionary, removed fields not supported by altair (eg, path/arc), and then called `.configure(**theme)`. For anyone looking for a more ready to use solution, this is what I have for the fivethirtyeight theme:

```python
theme = {
    "area": {"fill": "#30a2da"},
    "axis": {
        "domainColor": "#cbcbcb",
        "grid": True,
        "gridColor": "#cbcbcb",
        "gridWidth": 1,
        "labelColor": "#999",
        "labelFontSize": 10,
        "titleColor": "#333",
        "tickColor": "#cbcbcb",
        "tickSize": 10,
        "titleFontSize": 14,
        "titlePadding": 10,
        "labelPadding": 4,
    },
    "axisBand": {"grid": False},
    "legend": {
        "labelColor": "#333",
        "labelFontSize": 11,
        "padding": 1,
        "symbolSize": 30,
        "symbolType": "square",
        "titleColor": "#333",
        "titleFontSize": 14,
        "titlePadding": 10,
    },
    "line": {"stroke": "#30a2da", "strokeWidth": 2},
    "rect": {"fill": "#30a2da"},
    "range": {
        "category": [
            "#30a2da",
            "#fc4f30",
            "#e5ae38",
            "#6d904f",
            "#8b8b8b",
            "#b96db8",
            "#ff9e27",
            "#56cc60",
            "#52d2ca",
            "#52689e",
            "#545454",
            "#9fe4f8",
        ],
        "diverging": ["#cc0020", "#e77866", "#f6e7e1", "#d6e8ed", "#91bfd9", "#1d78b5"],
        "heatmap": ["#d6e8ed", "#cee0e5", "#91bfd9", "#549cc6", "#1d78b5"],
    },
    "point": {"filled": True, "shape": "circle"},
    "bar": {"binSpacing": 2, "fill": "#30a2da", "stroke": None},
    "title": {"anchor": "start", "fontSize": 24, "fontWeight": 600, "offset": 20},
}
alt.Chart().configure(**theme)
```

**Comment by joelostblom** [comment]
Since Altair 5.2, the functionality of Altair Saver is now available in Altair via the [vl-convert](https://github.com/vega/vl-convert) package. Most of the functionality has been available since 5.0, and [the main addition in 5.2 was PDF export](https://github.com/altair-viz/altair/pull/3244). See [the docs on how to save charts](https://altair-viz.github.io/user_guide/saving_charts.html#png-svg-and-pdf-format) for more details

We are going to archive this repo, so I'm closing all the open issues and PRs before doing so. Try out the new options for saving charts mentioned above and if you run into issues, please open an issue directly in the altair or vl-convert repo.
